File: benchmarks/loop.py
import time
print("Running Python Loop(1M)...")
sum = 0
i = 0
start = time.time()
while i < 1000000:
    sum = sum + i
    i = i + 1
end = time.time()
print(f"Result: {sum}")
print(f"Time: {end - start:.6f}s")

File: benchmarks/loop.rs
use std::time::Instant;

fn main() {
    println!("Running Rust Loop(1M)...");
    let mut sum: i64 = 0;
    let mut i: i64 = 0;
    let start = Instant::now();
    while i < 1_000_000 {
        sum += i;
        i += 1;
    }
    let duration = start.elapsed();
    println!("Result: {}", sum);
    println!("Time: {:.6}", duration.as_secs_f64());
}
